docs/changelog.rst

```diff
@@ -99,6 +99,10 @@ Detailed list of changes
 - A new action :ac:`scroll_prompt_to_bottom` to move the current prompt
   to the bottom, filling in the window from the scrollback (:pull:`4634`)
 
+- Add two special arguments ``@first-line-on-screen`` and ``@first-line-on-screen``
```

Comment:
Copy&Paste error.

Reply:
Thanks for pointing that out. Sorry. There are still mypy issues I haven't fixed in time. :(